src/utility/HCI.cpp

```diff
@@ -112,6 +112,15 @@ void HCIClass::poll(unsigned long timeout)
 
   while (HCITransport.available()) {
     byte b = HCITransport.read();
+	
+	if (sizeof(_recvBuffer) <= _recvIndex)
+	{
+	  _recvIndex = 0;
+	  if (_debug) {
+			_debug->println("_recvBuffer overflow");
```

Comment:
A bit of much indentation here, have you been using tabs instead of spaces?

Reply:
Oh, sorry.I used the tab.